index.html

```diff
@@ -34,6 +35,185 @@ <h1 class="text_title item-1">Vanessa Bonito</h1>
             </div>
             <div class="item-4"></div>
         </section>
+        <section class="grid-works">
+            <div class="div-title">
+                <h1 class="text_titleH2">Projects</h1>
+            </div>
+            <div class="container-work" id="project1">
+                <div class="img-work img-work-source1"></div>
+                <div class="div-title flex-work-title">
+                    <h1 class="text_titleH3">Project 1</h1>
```

Comment:
There should only be 1 heading per section or I would say per page. We can change to some other heading preferably `h3` or `h4`. What do you think? Same goes for the rest of the cards.